### PR TITLE
feat(lint): Update prettier

### PR DESCRIPTION
**Pull request by jeffkwoh**
Required to properly support nullish coalescence operator and optional chaining

Sister PR: https://github.com/getsentry/getsentry/pull/3481

### CODE REVIEW

**Review by billyvg — CHANGES_REQUESTED** [review]
Did you run prettier across our codebase?

**Comment by jeffkwoh** [comment]
@billyvg Thanks for the reminder, done.

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.jsx [hunk 1]
```diff
@@ -158,9 +158,7 @@ class GroupTagValues extends React.Component {
         <h3>
           {tagKey.key === 'user' ? t('Affected Users') : tagKey.name}
           <a
-            href={`/${orgId}/${group.project.slug}/issues/${group.id}/tags/${
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.jsx]: v nice

**Review by billyvg — COMMENTED** [review]
Did a quick glance, and it seems like this formatting is a nice improvement.  Anyone in @getsentry/app-frontend have any thoughts?

**Review by markstory — COMMENTED** [review]
@billyvg Generally looks good to me. The template string changes are 😍 . I was not a fan of all the line breaks that prettier added before.

**Review by billyvg — APPROVED** [review]

**Pull request merged**